### DISCUSSION TITLE
baize/developer/notebooks/baizectl

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4660] [Announcements]
# baize/developer/notebooks/baizectl

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/baize/developer/notebooks/baizectl.html

**Comment by panpan0000** [comment]
python代码为了支持咱们的baizectl分布式，应该添加哪些环境变量（DDP）？
DDP应该只需要这几个变量吧

MASTER_ADDR
MASTER_PORT
WORLD_SIZE
RANK
